File: lib/utils/geocoder/geocoder.util.dart
class GeocoderUtil {
  String? removeTag(String? raw) {
    if (raw == null) return null;

    if (!raw.contains("+")) {
      return raw;
    }

    var noPref = raw.substring(raw.indexOf("+"));
    bool shouldIngest = false;
    String address = "";

    for (int i = 0; i < noPref.length; i++) {
      if (shouldIngest) {
        address += noPref[i];
      }

      if (noPref[i] == " ") {
        shouldIngest = true;
      }
    }

    return address;
  }
}

File: web_player/src/utils/geocoder.util.ts
export const removeAddressTag: (r: string | undefined) => string = (raw) => {
    if (raw == undefined) return "";
    if (!raw.includes("+")) return raw;

    return raw.substring(raw.indexOf(" ") + 1);
}

export const extractMainAddress: (r: string) => string = (raw) => {
    return raw.substring(0, raw.indexOf(","));
}